features/steps/document.py

```diff
@@ -159,6 +159,12 @@ def when_I_add_an_even_page_section_to_the_document(context):
     context.section = context.document.add_section(WD_SECTION.EVEN_PAGE)
 
 
+@when('I assign bookmark = document.bookmarks.get("Target")')
+def when_I_assign_bookmark_eq_document_bookmarks_get_target(context):
+    document = context.document
```

Comment:
I would avoid these "one-time-use-only" assignments and just inline them like:

```python
context.bookmark = context.document.bookmarks.get("Target")
```

I know there are plenty of examples of doing this in other step implementations, but I'm thinking one line is better than two for these cases.

Same for the other steps below.

Reply:
Agreed, fixed